app/Reply.php

```diff
@@ -43,6 +43,12 @@ protected static function boot()
             $reply->owner->gainReputation('reply_posted');
         });
 
+        static::deleting(function ($reply) {
+            if ($reply->isBest()) {
+                $reply->thread->unsetBestReplay();
+            }
+        });
+
```

Comment:
Can someone explain this for me :) In the migrations we have foreign key to set best_reply_id to null on reply delete: https://github.com/JeffreyWay/council/blob/master/database/migrations/2017_03_15_175959_create_threads_table.php#L30-L34

On Laravel 5.5 [this test works](https://github.com/JeffreyWay/council/blob/master/tests/Feature/BestReplyTest.php#L47-L58), but on 5.6 it doesn't :( Why?